megamek/src/megamek/client/ui/swing/boardview/FieldofFireSprite.java

```diff
@@ -65,18 +63,35 @@ public class FieldofFireSprite extends MovementEnvelopeSprite {
     // individual sprite values
     private final Color fillColor;
     private final int rangeBracket;
+
     
     public FieldofFireSprite(BoardView boardView1, int rangeBracket, Coords l,
                              int borders) {
         // the color of the super doesn't matter
         super(boardView1, Color.BLACK, l, borders);
-        fillColor = new Color(fieldofFireColors[rangeBracket].getRed(), 
-                fieldofFireColors[rangeBracket].getGreen(),
-                fieldofFireColors[rangeBracket].getBlue(), 
-                borderOpac);
+        Color c = getFieldOfFireColor(rangeBracket);
+        fillColor = new Color(c.getRed(), c.getGreen(), c.getBlue(), borderOpac);
         this.rangeBracket = rangeBracket;
     }
-    
+
+    public static Color getFieldOfFireColor(int rangeBracket) {
+        // colors for Min,S,M,L,E ranges
+        switch (rangeBracket) {
+            case 0:
```

Comment:
If practical, can we re-use the existing range bracket constants rather than magic numbers?